.github/workflows/push-deploy-website.yml

```
@@ -32,15 +34,13 @@ jobs:
       # "on.push.branches" section at the top of this file
-      name: develop-postgres
+      name: develop
       url: https://docs-mobile.talawa.io
     steps:
       - uses: actions/checkout@v4
       - uses: actions/setup-node@v4
         with:
-          node-version: 22
-          cache: yarn
-          cache-dependency-path: 'docs/'
+          node-version: '20.x' 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)